plugins/ingest-attachment/src/test/resources/org/elasticsearch/ingest/attachment/test/sample-files/text-in-french.txt

```diff
@@ -0,0 +1 @@
+Allons enfants de la Patrie Le jour de gloire est arrivé. Contre nous de la tyrannie
```

Comment:
La Marseillaise! :) 

Thanks for keeping it here! But feel free to add "Das Lied der Deutschen" as well :p